docs/index.md

```diff
@@ -6,7 +6,8 @@ description: EthSigner is an open-source private transaction manager developed u
 
 EthSigner is an open-source, client-agnostic, Ethereum transaction signer developed under the Apache 2.0 license and written in Java. 
 
-EthSigner provides private key signing to separate key management from transaction validation. 
+EthSigner provides private key signing to separate key management from transaction validation. EthSigner supports [Hashicorp Vault](Using-EthSigner/Hashicorp.md) 
```

Comment:
I know we've always had this text, but wondering if we care enough to change it?

> Ethsigner is able to sign transactions using keys managed either locally (on the file system) or within cloud-based services (Azure and Hashicorp). The use of cloud-based key management systems effectively separates business logic from key storage.